### DISCUSSION TITLE
Dnscrypt-proxy & Google

### DISCUSSION THREAD

**Discussion by pashamcr** [DNSCrypt/dnscrypt-proxy#2030] [Q&A]
Windows 10, Firefox.
When switching to Dnscrypt-proxy, in the browser when using Google search, the safe mode is enabled in the Google search settings, which cannot be disabled. YouTube also does not work correctly: not all links open, comments are hidden. Is it possible to get around this?

**Comment by jedisct1** [comment]
Either you enabled Google Safe Search in `cloaking-rules.toml`, or you're using a DNS resolver whose purpose is to do parental control.

**Reply by pashamcr** [reply]
Thanks for the tip, I used cloaking-rules without looking into the config file.
